state-chain/pallets/cf-environment/src/tests.rs

```diff
@@ -53,7 +53,7 @@ fn test_btc_utxo_selection() {
 		add_utxo_amount(dust_amount);
 
 		// select some utxos for a tx
-		const EXPECTED_CHANGE_AMOUNT: crate::BtcAmount = 24770;
+		const EXPECTED_CHANGE_AMOUNT: crate::BtcAmount = 24680;
```

Comment:
why is this the expected amount? could we include a comment here explaining this, as well as the other static values